draft-ietf-teep-architecture.md

```diff
@@ -288,6 +281,13 @@ The following terms are used:
    Trusted Components.  Trusted OS code or trusted firmware can also be
    expressed as Trusted Components that a TA depends on.
 
+  - Trusted Component Signer: An entity that signs a Trusted Component with
+    a key that a TEE will trust.  The signer might or might not be the
+    same entity as the TA Developer.  For example, a TA might
+    be signed (or re-signed) by a Device Administrator if the TEE will
+    only trust the Device Administrator.  A TA might also be encrypted,
```

Comment:
"A TA might" --> "A Trusted Component". Also remove one of the two spaces before it.